src/components/Banner/examples/basic.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import Banner from '../Banner';
+import classNames from 'classnames';
+
+const {
+	PropTypes: {
+		number,
+		string
+	}
+} = React;
+
+
+const InfoIcon = React.createClass({
+	propTypes: {
+		className: string,
+		color: string,
+		size: number,
```

Comment:
good point.  this is just a place holder to see what it would look like if a user would put a custom icon.